cblite/llm/OpenAI.cc

```diff
@@ -22,16 +22,12 @@ using namespace std;
 using namespace fleece;
 using namespace litecore;
 
-alloc_slice OpenAI::run(const string& restBody, C4Error error) {
+alloc_slice OpenAI::runSubclass(const string& restBody, C4Error error) {
```

Comment:
This is kind of flipped on its head from what I would recommend.  A method called `runSubclass` is not a good look because it is leaking lots of implementation details about how this is put together. 

Rather, a better way would be to keep this method as `run` so that as far as outside callers are concerned, they are just calling a method called `run` which makes sense (otherwise the question is "why am I calling `runSubclass`?) and have a protected virtual method that returns the response object that run can call.